### ISSUE TITLE
Is this repo have paper  arXiv link?

### ISSUE THREAD

**Issue by GewelsJI** [koutilya40192/BiConvLSTM_Violence_Detection#1]

**Comment by j-alex-hanson** [comment]
Yes, here's the [link](https://openaccess.thecvf.com/content_ECCVW_2018/papers/11130/Hanson_Bidirectional_Convolutional_LSTM_for_the_Detection_of_Violence_in_Videos_ECCVW_2018_paper.pdf).
